TASVideos.WikiEngine/MakeBracketed.cs

```diff
@@ -258,17 +258,19 @@ internal static INode MakeLink(int charStart, int charEnd, string text, INode ch
 		return new Element(charStart, "a", attrs, [child]) { CharEnd = charEnd };
 	}
 
-	private static Element MakeImage(int charStart, int charEnd, string[] pp, int index, out bool unusedParams)
+	private static Element MakeImage(int charStart, int charEnd, IEnumerable<string> pp, out bool unusedParams)
 	{
 		unusedParams = false;
+		var iter = pp.GetEnumerator();
```

Comment:
I don't think downgrading this from array to IEnumerable is an improvement here.  It makes it more extensible, but this is only used in specific circumstances.  In those cases, I generally prefer to be specific with the types that are being used.  Using iterators, imo, hurts readability compared to a standard for loop, and I don't see any performance benefits

Reply:
Dropped